cmd/quota-set.go

```diff
@@ -32,7 +32,7 @@ import (
 var quotaSetFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "size",
-		Usage: "set a hard quota, disallowing writes after quota is reached",
+		Usage: "set a quota, disallowing writes after quota is reached",
```

Comment:
```suggestion
		Usage: "Set a quota or usage limit for a bucket. Depends on the scanner for determining usage."
```

Maybe - a little weird because we don't talk about the scanner, but I think it's not terrible either.

Reply:
Is there a practical limit on how long these descriptions should be? Maybe
```suggestion
		Usage: "set a quota, disallowing writes if the object scanner determines the quota has been exceeded.",
```

The text added yesterday does cover the scanner at least. This is what it currently looks like:
```
$ ./mc --dp quota set
NAME:
  mc quota set - set bucket quota

USAGE:
  mc quota set TARGET [--size QUOTA]

QUOTA
  Quota accepts human-readable case-insensitive number.
  Suffixes such as "k", "m", "g" and "t" referring to the metric units KB,
  MB, GB and TB respectively. Adding an "i" to these prefixes, uses the IEC
  units, so that "gi" refers to "gibibyte" or "GiB". A "b" at the end is
  also accepted. Without suffixes the unit is bytes.

  The MinIO object scanner checks a bucket's quota each time it is scanned.
  If the scanner determines a bucket has met or exceeded its quota, MinIO
  rejects subsequent object write requests until the scanner determines the
  bucket no longer exceeds its quota.

FLAGS:
  --size value                  set a hard quota, disallowing writes after quota is reached
  --config-dir value, -C value  path to configuration folder (default: "/Users/andrealongo/.mc") [$MC_CONFIG_DIR]
  --quiet, -q                   disable progress bar display [$MC_QUIET]
  --disable-pager, --dp         disable mc internal pager and print to raw stdout [$MC_DISABLE_PAGER]
  --no-color                    disable color theme [$MC_NO_COLOR]
  --json                        enable JSON lines formatted output [$MC_JSON]
  --debug                       enable debug output [$MC_DEBUG]
  --insecure                    disable SSL certificate verification [$MC_INSECURE]
  --limit-upload value          limits uploads to a maximum rate in KiB/s, MiB/s, GiB/s. (default: unlimited) [$MC_LIMIT_UPLOAD]
  --limit-download value        limits downloads to a maximum rate in KiB/s, MiB/s, GiB/s. (default: unlimited) [$MC_LIMIT_DOWNLOAD]
  --help, -h                    show help
  
EXAMPLES:
  1. Set hard quota of 1gb for a bucket "mybucket" on MinIO.
     $ mc quota set myminio/mybucket --size 1GB
```

Reply:
If it's very wide I guess it can clip in the CLI view, but that's not the end of the world either. Your edit works for me.

```
set a hard quota on the bucket based on scanner usage measurements
```

Brings back `hard` but at least folds the scanner into it.